### PR TITLE
fix(null checks): Better Null checking

### PR DESCRIPTION
**Pull request by nmccready**

### CODE REVIEW

File: src/services/leafletHelpers.js [hunk 1]
```diff
@@ -14,21 +14,15 @@ angular.module('ui-leaflet').service('leafletHelpers', function($q, $log) {
...
-        obj = object;
-        pathStr.split('.').forEach(function(value) {
-            if (obj) {
-                obj = obj[value];
-            }
-        });
+    var _getObjectValue = function (object, path){
+        if(!object) return;
+        path = path.split('.');
+        var obj = object[path.shift()];
```
Comment (nmccready) [inline src/services/leafletHelpers.js]: Using while and shift should be quicker then forcibly iterating through everything via forEach.

File: dist/ui-leaflet.js [hunk 2]
```diff
@@ -230,14 +230,15 @@ angular.module('ui-leaflet').factory('leafletBoundsHelpers', ["leafletLogger", "
...
+        get = leafletHelpers.get,
         createLayer = leafletLayerHelpers.createLayer,
         _controls = {},
         errorHeader = leafletHelpers.errorHeader + ' [Controls] ',
         $log = leafletLogger;
 
     var _controlLayersMustBeVisible = function(baselayers, overlays, mapId) {
         var defaults = leafletMapDefaults.getDefaults(mapId);
-        if(!defaults.controls.layers.visible) {
+        if(!get(defaults, 'controls.layers.visible')) {
```
Comment (nmccready) [inline dist/ui-leaflet.js]: This kinda stuff is where I much prefer coffeescript... anyway using `get` to get a value as undefined regardless how nested.

**Comment by nmccready** [comment]
@elesdoar

File: src/services/leafletHelpers.js (hunk 1 above)
Comment (elesdoar) [inline src/services/leafletHelpers.js]: Good...

**Comment by elesdoar** [comment]
LGTM!

**Pull request merged**